src/test/java/net/sourceforge/plantuml/servlet/TestMultipageUml.java

```diff
@@ -0,0 +1,199 @@
+        final URL url = new URL(getServerUrl() + "/png/" + ENCODED);
+        final URLConnection conn = url.openConnection();
+        // Analyze response
+        // Verifies the Content-Type header
+        assertEquals(
+            "Response content type is not PNG",
+            "image/png",
+            conn.getContentType().toLowerCase()
+        );
+        // Get the image and verify its size
+        byte[] inMemoryImage = getContentAsBytes(conn);
+        int diagramLen = inMemoryImage.length;  // 4196
+        assertTrue(diagramLen > 3000);
+        assertTrue(diagramLen < 5000);
+    }
+
+    /**
+     * Verifies that an multipage diagram renders correct given index (SVG).
+     */
+    public void testSvgIndexPage() throws IOException {
+        final URL url = new URL(getServerUrl() + "/svg/1/" + ENCODED);
+        final URLConnection conn = url.openConnection();
+        // Analyze response
+        // Verifies the Content-Type header
+        assertEquals(
+            "Response content type is not SVG",
+            "image/svg+xml",
+            conn.getContentType().toLowerCase()
+        );
+        // Get the content and verify its size
+        String diagram = getContentText(conn);
+        int diagramLen = diagram.length();  // 5834
+        assertTrue(diagramLen > 5500);
+        assertTrue(diagramLen < 7000);
+    }
+
+    /**
+     * Verifies that an multipage diagram renders correct even if no index is specified (SVG).
+     */
+    public void testSvgIndexPageNoIndex() throws IOException {
+        final URL url = new URL(getServerUrl() + "/svg/" + ENCODED);
+        final URLConnection conn = url.openConnection();
+        // Analyze response
+        // Verifies the Content-Type header
+        assertEquals(
+            "Response content type is not SVG",
+            "image/svg+xml",
+            conn.getContentType().toLowerCase()
+        );
+        // Get the content and verify its size
+        String diagram = getContentText(conn);
+        int diagramLen = diagram.length();  // 2970
+        assertTrue(diagramLen > 1500);
+        assertTrue(diagramLen < 4000);
+    }
+
+    /**
+     * Verifies that an multipage diagram renders correct given index (AsciiArt).
+     */
+    public void testAsciiArtIndexPage() throws IOException {
+        final URL url = new URL(getServerUrl() + "/txt/1/" + ENCODED);
+        final URLConnection conn = url.openConnection();
+        // Analyze response
+        // Verifies the Content-Type header
+        assertEquals(
+            "Response content type is not TEXT PLAIN or UTF-8",
+            "text/plain;charset=utf-8",
+            conn.getContentType().toLowerCase()
+        );
+        // Get the content and verify its size
+        String diagram = getContentText(conn);
+        int diagramLen = diagram.length();
+        assertNotEquals(0, diagramLen);
+        // BUG/Missing Feature: plantuml renders always whole AsciiArt diagram
```

Comment:
Can somebody please verify, that this is a missing feature or bug in the plantuml library?